Updating constitution and Bylaws, added absentee voting (#30)
IronRiders/Constitution: Member Expectations.md

> # Member expectations

Reviewer: ```suggestion
# Member Expectations

```

> -Must communicate to department leads what days they will be there<br/>

Reviewer: ```suggestion
* Must communicate to department leads what days they will be there
```

> -There is a no-tolerance policy for all discrimination including but not limited to homophobia, racism, sexism, transphobia, religious discrimination, etc.<br/>

Reviewer: ```suggestion
* There is a no-tolerance policy for all discrimination including but not limited to homophobia, racism, sexism, transphobia, or religious discrimination
```

> -Must contribute positively<br/>

Reviewer: ```suggestion
* Must contribute positively
```

> -Must have a willingness to learn<br/>

Reviewer: ```suggestion
* Must have a willingness to learn
```

> -Respect the leads, mentors, and other members<br/>

Reviewer: ```suggestion
* Respect the leads, mentors, and other members
```

> -Be willing to teach others<br/>

Reviewer: ```suggestion
* Be willing to teach others
```

> -Failure to follow these rules will result in a conversation with the leads and/or Mr. Ruff<br/>

Reviewer: ```suggestion
* Failure to follow these rules will result in a conversation with the leads and/or coaches
```

> -In extreme cases, failure to follow these expectations will result in a suspension<br/>

Reviewer: ```suggestion
* In extreme cases, failure to follow these expectations will result in a suspension
```

> -Must fill out team forms<br/>

Reviewer: ```suggestion
* Must fill out team forms
```
